tests/test-run.sh

```diff
@@ -80,6 +80,28 @@ assert_file_has_content runtime-fpi "^runtime=runtime/org\.test\.Platform/$ARCH/
 
 ok "run a runtime"
 
+if [ -f /etc/os-release ]; then
+    run_sh org.test.Platform cat /run/host/os-release >os-release
+    (cd /etc; md5sum os-release) | md5sum -c
```

Comment:
I wanted to say use the assert_file_* helpers to compare for equality as they print better errors, but we don't seem to have an file equal helper atm...

Reply:
What would a file-equal helper do? Exit zero if identical, or nonzero with diagnostic output if different? If that's what you want, that's just diff(1).